commerce-service/src/main/java/com/liferay/commerce/internal/starter/CommerceRegionsStarterBase.java

```diff
@@ -0,0 +1,92 @@
+package com.liferay.commerce.internal.starter;
+
+import com.liferay.commerce.model.CommerceCountry;
+import com.liferay.commerce.service.CommerceCountryLocalService;
+import com.liferay.commerce.service.CommerceRegionLocalService;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.util.StringUtil;
+
+/**
+ * @author Riccardo Alberti
+ */
+public class CommerceRegionsStarterBase {
```

Comment:
Should be called `BaseCommerceRegionsStarter`, and in my opinion it should be `abstract` and implement `CommerceRegionsStarter`, to make it clearer how this class is intended to be used.